src/core/include/iDynTree/Core/Rotation.h

```diff
@@ -271,6 +296,19 @@ namespace iDynTree
          */
         static Rotation Identity();
 
+        /**
+         * Construct a rotation matrix from the given quaternion representation
+         *
+         * The quaternion is expected to be ordered in the following way:
+         * - s \in \mathbb{R} the constituent of the quaterion
```

Comment:
It's written in the reference where I took the formula.
Should I change it to real part?